Modifications issues #82

AmndneFR opened the pull request.

Commit:
Modifications issues #82

Jerome-CM commented on src/main/java/com/bugspointer/service/implementation/MailService.java:
```diff
@@ -31,6 +31,8 @@ public class MailService {
     @Value("${mail.password}")
     private String password;
 
+    private static final String adresse = "http://bugspointer.com/";
```
A changer par ADDRESS

Une constante se déclare en majuscule
Utiliser les constantes est bénéfique pour deux principales raisons. La première est que cela évitera d'accidentellement modifier la valeur d'une variable qui ne devrait pas changer. Le compilateur veillera en effet à ce que cela n'arrive pas. La deuxième est que cela permet au compilateur de faire des optimisations pouvant accélérer l'exécution du code.

Jerome-CM commented on src/main/resources/templates/download/modal.html:
```diff
@@ -28,10 +28,9 @@
-              <button class="bugspointer-popup-btn-submit" type="button" id="bugspointer-popup-btn-pointer">Pointer un bug</button>
-              <p id="bugspointer-popup-pointer-bug-text">Merci de sélectionner la ligne contenant le bug en cliquant sur le bouton si dessus</p>
+              <button class="bugspointer-popup-btn-submit" type="button" id="bugspointer-popup-btn-pointer" title="Cliquez-ici avant d'envoyer le formulaire">Pointer le bug</button>
+              <p class="bugspointer-popup-pointer-bug-text" id="bugspointer-popup-pointer-bug-text">Merci de sélectionner la ligne contenant le bug en cliquant sur le bouton si dessus</p>
```
Merci de sélectionner l'endroit  contenant le bug grâce au bouton si dessus

Ce n'est pas forcement une ligne qui sera bugger mais peut être une image ou un document

Commit:
Modifications issues #82
  - Ajout du domaine dans Account
  - Ajout du domaine pour valider l'envoi d'un bug
  - Ajout du timer et boolean pour activer ou non le bouton d'envoi
  - Test CSS pour arreter les gif (sur 'au-revoir' n'a pas fonctionné)

Jerome-CM committed:
stop animation gif
